### ISSUE TITLE
Error: Invalid color: none

### ISSUE THREAD

**Issue by andry1001** [henninghall/react-native-date-picker#258]
On Android, updated from 3.02 -> 3.2.5.

```
<DatePicker
    date={new Date(props.date)}
    onDateChange={props.onChange}
    style={styles.datePicker}
    locale={I18n.locale}
    textColor={colors.primaryTextColor}
    mode={props.mode ? props.mode : "datetime"}
    is24hourSource='locale'
    fadeToColor='none'
 />
```

`fadeToColor='none'` throws "Invalid color: none" now and I couldn't find a way to remove the fade color. Seems like has something to do with `v3.1.0` and seems to me from documentation that it should still work.

**Comment by VivienTournie** [comment]
Same error after updating to the last version : "react-native-date-picker": "^3.2.5"

**Comment by jmoliugp** [comment]
@andry1001 try not passing any value to `fadeToColor` prop. It worked for me.

**Comment by andry1001** [comment]
> 
> 
> @andry1001 try not passing any value to `fadeToColor` prop. It worked for me.


Works with `androidVariant="nativeAndroid"`, which I amm using right now, but with  `androidVariant="iosClone"` it fades to white on Android.

**Comment by henninghall** [comment]
Fixed in `v3.2.6 `
